### PR TITLE
enhance(plus): finishing touches for collectionsV2

### PR DESCRIPTION
**Pull request by fiji-flo**
- no more hero border
- banner for multiple collections
- article count links to collection
- updated default collection copy
- updated core plan

### CODE REVIEW

File: client/src/banners/active-banner.tsx [hunk 1]
```diff
@@ -137,6 +139,36 @@ function PreviewFeaturesBanner({ onDismissed }: { onDismissed: () => void }) {
...
+  const sendCTAEventToGA = useSendCTAEventToGA();
+  const gleanClick = useGleanClick();
+  const onDismissedWithGlean = () => {
+    gleanClick(BANNER_MULTIPLE_COLLECTIONS_DISMISSED);
+    onDismissed();
+  };
+  return (
+    <Banner id={bannerId} onDismissed={onDismissedWithGlean}>
+      <p className="mdn-cta-copy">
+        You can now create multiple collections.{" "}
```
Comment (Guyzeroth) [inline client/src/banners/active-banner.tsx:157]: I tried the  copy 

"We've added support for Multiple Collections! Check out your **Collections page** to see what's new!"

File: client/src/plus/collections/v2/index.scss [hunk 2]
```diff
@@ -19,7 +19,6 @@
           rgba(232, 8, 64, 0.3) 100%
         );
         /* Light/Color/Icon/Critical */
-        border: 1px solid var(--text-primary-red);
```
Comment (Guyzeroth) [inline client/src/plus/collections/v2/index.scss:22]: rip border

File: client/src/plus/collections/v2/index.tsx [hunk 3]
```diff
@@ -46,11 +46,21 @@ function Overview() {
...
-              Save and group your favorite MDN articles <br />
-              to easily find them later on.
+              Save and group your favorite MDN articles to easily find them
+              later on. <br />
+              <a
+                rel="noreferrer noopener"
+                target="_blank"
+                href="https://www.surveygizmo.com/s3/6988450/Feature-Preview-User-Feedback-Multiple-Collections"
+              >
+                This is a new feature. Give us some feedback.
```
Comment (Guyzeroth) [inline client/src/plus/collections/v2/index.tsx:62]: We'd love to hear your feedback!

File: client/src/banners/index.tsx [hunk 4]
```diff
@@ -11,21 +11,17 @@ import { CRUD_MODE } from "../env";
...
 export function Banner() {
   const userData = useUserData();
-  const currentBannerId: BannerId | null =
-    userData?.subscriptionType &&
-    [SubscriptionType.MDN_PLUS_10M, SubscriptionType.MDN_PLUS_10Y].includes(
-      userData?.subscriptionType
-    )
-      ? BannerId.PREVIEW_FEATURES
-      : BannerId.PLUS_LAUNCH_ANNOUNCEMENT;
+  const currentBannerId: BannerId | null = userData?.isAuthenticated
```
[the remark this answers is not in the document]
Comment (fiji-flo) [reply client/src/banners/index.tsx:22]: That's correct. However we set it to `null` if we disable all banners and don't wanna change the type annotation all the time.

**Pull request merged**